NotesApi/V1/Gateways/NotesDbGateway.cs

```diff
@@ -36,15 +35,27 @@ public async Task<PagedResult<Note>> GetByTargetIdAsync(GetNotesByTargetIdQuery
             var dbNotes = new List<NoteDb>();
             var table = _dynamoDbContext.GetTargetTable<NoteDb>();
 
+            var filterExpression = new Expression();
+            filterExpression.ExpressionAttributeNames.Add("#t", "targetId");
```

Comment:
I would accept a list of filters to the call, and then construct a filter that way (so pass in the category to filter on instead of hard coding it here)

Reply:
Agree